components/Careers/AllJobsPage.vue

```diff
@@ -0,0 +1,427 @@
+<template>
+    <body class="font-nunito text-base text-black dark:text-white dark:bg-slate-900">
```

Comment:
Update component name: Page should not be used in component name

Reply:
you can directly put all this code in page folder, why to make component like this ?

Reply:
I had to scope the css for this component only.